lib/reek/spec/spec.rb

```diff
@@ -1,6 +1,6 @@
-require_relative 'spec/should_reek'
-require_relative 'spec/should_reek_of'
-require_relative 'spec/should_reek_only_of'
+require_relative 'should_reek'
+require_relative 'should_reek_of'
+require_relative 'should_reek_only_of'
```

Comment:
I would prefer to keep this file at `reek/spec.rb`, for two reasons:

* When a namespace *foo* is introduced, I prefer the idiom of having `foo.rb` and `foo/some_class.rb` over `foo/foo.rb` or `foo/base.rb`.
* Users will now have to `require 'reek/spec/spec'`, which is a breaking change and looks odd.